src/app/products/services/localstorage.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import Product from '../interfaces/product.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LocalStorageService {
+  public setData(items: Map<Product, number>): void {
+    localStorage.setItem('cart', JSON.stringify([...items]));
```

Comment:
`cart` looks like something that should const/enum, not just string value.
Also notive that `LocalStorageService` could work with different keys in future